fe/fe-core/src/main/java/org/apache/doris/planner/external/ExternalFileScanNode.java

```diff
@@ -253,45 +264,40 @@ private void buildScanRange() throws UserException, IOException {
         String filePath = ((FileSplit) inputSplits[0]).getPath().toUri().getPath();
         String fsName = fullPath.replace(filePath, "");
         hdfsParams.setFsName(fsName);
-        List<String> partitionKeys = new ArrayList<>();
-        for (FieldSchema fieldSchema : hmsTable.getRemoteTable().getPartitionKeys()) {
-            partitionKeys.add(fieldSchema.getName());
-        }
+
 
         for (InputSplit split : inputSplits) {
             FileSplit fileSplit = (FileSplit) split;
 
             TScanRangeLocations curLocations = newLocations(context.params);
```

Comment:
Here you create `TScanRangeLocations` for every file?
That would be too many `TScanRangeLocations`.